doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex

```diff
@@ -183,6 +183,468 @@ \subsubsection{Outputs}\label{outputs-015}
+
+This alpha field specifies if the coil will operate to meet a latent load calculated from the air flow rate through the coil system, coil system entering air humidity ratio and coil system outlet node (control node) maximum humidity ratio setpoint. There are two valid choices, Yes or No. If Yes, the coil will run if there is a latent load. If both a sensible and latent load exist, the system will operate to maintain the temperature set point and then activate dehumidification control if needed. When only a latent load exists, the system will operate to meet the maximum humidity ratio set point and may require the use of a heating coil and heating coil outlet node air temperature set point manager downstream of this cooling coil to maintain the temperature set point. If No, the coil will not run if there is only a latent load. The default is No.
+
+
+\paragraph{Field: Minimum Air To Water Temperature Offset [deltaC]}\label{field-minimum-air-to-water-temperature-offset}
+
+The coil system will turn ON as required when coil entering air temperature is above coil entering water temperature by more than the amount of this offset [deltaC]. To model a waterside economizer connected to condenser loop increase offset as desired. Default is 0.
+
+
+Following is an example input for a coil system cooling water.
+
+\begin{lstlisting}
+
+  CoilSystem:Cooling:Water,
+    Unitary-Free-Cooling,    !- Name
+    Mixed Air Node,          !- Air Inlet Node Name
+    FreeClgCoil OutletNode,  !- Air Outlet Node Name
+    ,                        !- Availability Schedule Name
+    Coil:Cooling:Water,      !- Cooling Coil Object Type
+    Free Cooling Coil,       !- Cooling Coil Name
+    CoolReheat,              !- Dehumidification Control Type
+    Yes,                     !- Run on Sensible Load
+    Yes,                     !- Run on Latent Load
+    3.0;                     !- Minimum Air To Water Temperature Offset
+	
+\end{lstlisting}
+
+
+\subsubsection{Outputs}\label{outputs}
+
+Following are the list of possible output variables from this coil model:
+
+\begin{itemize}
+\item HVAC, Average, Coil System Water Part Load Ratio {[]}
+\item HVAC, Average, Coil System Water Total Cooling Rate {[}W{]}
+\item HVAC, Average, Coil System Water Sensible Cooling Rate {[}W{]}
+\item HVAC, Average, Coil System Water Latent Cooling Rate {[}W{]}
+\item HVAC, Average, Coil System Water Control Status {[]}
+\end{itemize}
+
+
+\paragraph{Coil System Water Part Load Ratio {[]}}\label{coil-system-water-part-load-ratio}
+
+This output variable is the ratio of the sensible cooling load to the current full cooling capacity of the coil system. This variable reports the average load met as a fraction of the full coil capacity during the system timestep. If the ratio is 0.0, then there is no cooling load, else if the ratio is 1.0, then the load met is equal to the coil system full capacity.
+
+\paragraph{Cooling Coil Total Cooling Rate {[}W{]}}
```

Comment:
Just noticed, this header and the next two should start with "Coil System Water ..."

Reply:
OK. I will fix it.